fly/commands/pipeline_overview.go

```diff
@@ -0,0 +1,147 @@
+package commands
+
+import (
+	"os"
+
+	"github.com/concourse/concourse/atc"
+	"github.com/concourse/concourse/fly/commands/internal/displayhelpers"
+	"github.com/concourse/concourse/fly/rc"
+	"github.com/concourse/concourse/fly/ui"
+	"github.com/fatih/color"
+)
+
+type PipelineOverviewCommand struct {
+	Pipeline string `short:"p" long:"pipeline" required:"true" description:"Get resources in this pipeline"`
+	Json     bool   `long:"json" description:"Print command result as JSON"`
+}
```

Comment:
What do you think about adding flags to allow users to only print either the jobs or resources? Maybe `-r`/`--resources` prints only the resources and `-j`/`--jobs` prints only the jobs.

Reply:
I thought about this, it would almost make sense though, to then get rid of `fly jobs` and `fly resource` and have them all be part of this command.  Is that too big of a change at once though

Reply:
That would be quite a bigger diff. I'm fine just leaving it as-is